### PR TITLE
Document pre-commit git hook usage

### PR DESCRIPTION
**Pull request by dlukeomalley**
Addresses #92 and supports 0.7 content.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,13 +135,36 @@ jobs:
...
+ ```
+ 
+Otherwise, address the issues or archive them with `bento archive`.
+
+If you need help setting up Bento with another CI provider please [open an issue](https://github.com/returntocorp/bento/issues/new?template=feature_request.md). Documentation PRs welcome if you set up Bento with a CI provider that isn't documented here!
+
+### Running Bento as a Git Hook
+Bento can automatically analyze your staged files when `git commit` is run. Configured as a Git pre-commit hook, Bento ensures every commit to your project is vetted and that no new issues have been introduced to the codebase.
+
+To install Bento as a a Git hook:
```
Comment (pabloest) [inline README.md]:
```suggestion
To install Bento as a Git hook:
```

File: README.md [hunk 2]
```diff
@@ -127,13 +135,36 @@ jobs:
...
+
+```bash
+$ bento install-hook
+```
+
+If Git hooks ever incorrectly block your commit you can always skip them by passing the `--no-verify` flag at commit-time (use this sparingly):
```
Comment (pabloest) [inline README.md]:
```suggestion
If Git hooks ever incorrectly block your commit, you can skip them by passing the `--no-verify` flag at commit-time (use this sparingly):
```

File: README.md [hunk 3]
```diff
@@ -127,13 +135,36 @@ jobs:
...
+
+```bash
+$ git commit --no-verify
+```
 
-Please [open an issue](https://github.com/returntocorp/bento/issues/new?template=feature_request.md) if you need help setting up Bento with another CI provider. If you set up Bento with your provider of choice, we’d appreciate a PR to add instructions here! 
+Bento's Git hook can save the round-trip time involved with fixing a failed build if you're using [Bento in CI](#running-bento-in-ci).
```
Comment (pabloest) [inline README.md]:
```suggestion
Bento’s Git hook can save the round-trip time involved with fixing a failed build if you’re using [Bento in CI](#running-bento-in-ci). 
```

File: README.md [hunk 4]
```diff
@@ -127,13 +135,36 @@ jobs:
 Otherwise, you can simply install and run Bento in CI with the following commands:
 
 ```bash
-pip3 install bento-cli && bento --version
-bento --agree --email <YOUR_EMAIL> check
+$ pip3 install bento-cli && bento --version
```
Comment (pabloest) [inline README.md]: Curious, why is the `bento --version` needed here?

File: README.md [hunk 5]
```diff
@@ -35,6 +35,8 @@ Install, configure, and adopt Bento in seconds. Runs 100% locally.
   <span> · </span>
   <a href="#running-bento-in-ci">CI/CD</a>
   <span> · </span>
+  <a href="#running-bento-as-a-git-hook">Git Hook</a>
```
Comment (pabloest) [inline README.md]: An idea to slightly simplify the top menu would be to replace CI/CD and Git Hook with “Automation” and then create an Automation section below at the same level (`##`) as Installation, Usage, Help & Community. Automation would have two subsections, CI/CD and Git Hook. What do you think?

File: README.md (hunk 4 above)
Comment (dlukeomalley) [reply README.md]: Not needed, but it ensures in your log you know what version of Bento ran

Comment (pabloest) [reply README.md]: 👍

**Pull request merged**